database/header.go

```diff
@@ -14,31 +14,31 @@ func makeProtoHeader(h *decoder.HEP, bb *bytebufferpool.ByteBuffer) string {
 	bb.Reset()
 	bb.WriteString(`{`)
 	bb.WriteString(`"protocolFamily":`)
-	bb.WriteString(strconv.FormatUint(uint64(h.Version), 10))
+	bb.WriteString(strconv.FormatUint(uint64(h.CapturedIPVersion), 10))
```

Comment:
As for the Parsed prefix for the SIP struct why not remove Captured. I see no value in this redundant naming.

Reply:
Context. It's the packet ingested at the capture node. Right?

Reply:
If you look closely like here https://github.com/sipcapture/heplify-server/blob/master/cmd/heplify-server/heplify-server.go#L90

you will notice that heplify-server is designed to run multiple capture routines at the same time. Currently we have only HEP but I have already designed way more than that to be implemented in the future. So Captured is pretty contextless in that case. Some like HEP would make more sense, but than you will notice whats that h. could it stand for HEP? Bingo! :D

Reply:
Some more background information for you:

http://research.swtch.com/names
https://talks.golang.org/2014/names.slide
https://www.lysator.liu.se/c/pikestyle.html

Reply:
I had no idea that's what 'h' was. :) It's of course great that it's extensible. There are almost no comments in the code, nor clear variable names. Adding the two letters 'ep', many other things start to become clearer for me. Just a suggestion.

Reply:
> Some more background information for you:
> 
> http://research.swtch.com/names
> https://talks.golang.org/2014/names.slide
> https://www.lysator.liu.se/c/pikestyle.html

Second slide:
```
Names matter

Readability is the defining quality of good code.
Good names are critical to readability. 
```

Agree with short variable names in local functions.

Reply:
keyword: local (as opposed to global)

Reply:
Let's consider what these two theoretical variable namings might impart:

h.Version --> Version of HEP? Version of... what?
hep.CapturedIPVersion --> Clear. Unambiguous.

Reply:
Don't get me wrong, I'm totally with you that the naming can be improved ;) But something like
hep.CapturedIPVersion 
hurts my eyes ;)
If you want make it clear thats an input or captured just use hc for hepcaptured like

hc.IPVersion Clear. Unambiguous. Perfect :D

Reply:
Less clear. hc would be the object which represents the ingress HEP packet. HEP itself encapsulates the Captured payload. HEP is not itself a captured payload (at least not yet) - that would be a double encapsulation.

Reply:
Well I would say it's a matter of taste if you use h, hc or hep for the object name. I'm ok with all of them. But that "Captured" prefix makes in a compiled language were you have powerful ide's and editors no sense to me.

Reply:
OK - Captured says where the data comes from. HEP can contain data (captured, encapsulated), and itself have properties (like version, size, maybe others). The idea being that the past tense 'ed' adds temporal value to the understanding. 

What is an acceptable middle-ground?  Please don't use 'Got'.

Reply:
Well if you want to do it right you would need to change the current HEP struct. First rename the HEP and use something meaningful. Move all HEP related properties from the renamed Struct into a HEP struct. Embed that new HEP only struct into the renamed one.

For now I don't think that it's necessary because the project is small. Mby in the future. So for now I think I would accept following:

keep SIP, remove the Captured prefix. We can still do this when the project grows and I would be happy to discuss it with you then.

Reply:
You are a talented defender of the code-base ;)

For the struct: I don't want to abstract anything.

Reply:
If you really want to see talented defender of code-bases you can try to make such a pr to Prometheus ;)